core/tools.js

```diff
@@ -1518,6 +1518,71 @@
 		},
 
 		/**
+		 * Converts hexstring to base64 coode.
```

Comment:
This markdown does not get converted at all:
![image](https://user-images.githubusercontent.com/5353898/30712162-60ddacb8-9f0b-11e7-9829-c7187582dd3d.png)
Take the time and set up the ckeditor-docs builder, and preview any docs changes before pushing the code.

Reply:
Reason for the above is probably no new line before the ordered list.